src/emc/kinematics/genhexkins.c

```diff
@@ -11,7 +11,7 @@
 *    
 * Copyright (c) 2004 All rights reserved.
 *
-* Last change: 2014.12.22
+* Last change: 2016.08.04
```

Comment:
copyright notices and dates probably have legal meaning, don't remove copyright notices.  thanks for the additional commit in the series.